jsonrpc/README.md

```
@@ -69,6 +69,13 @@ func main() {
 	log.Println("Response:", resp)
 	log.Println("User:", user)
 
+	// Invoke a remote method with custom context middlewares
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)